src/app/public/modules/ag-grid/fixtures/ag-grid-row-delete.component.fixture.ts

```diff
@@ -0,0 +1,114 @@
+import {
+  Component,
+  OnInit,
+  ViewEncapsulation
+} from '@angular/core';
+
+import {
+  SkyAgGridService
+} from '../ag-grid.service';
+
+import {
+  GridOptions
+} from 'ag-grid-community';
+
+import {
+  SKY_AG_GRID_DATA,
+  SKY_AG_GRID_LONG_DATA
+} from './ag-grid-data.fixture';
+
+import {
+  SkyAgGridRowDeleteCancelArgs
+} from '../types/ag-grid-row-delete-cancel-args';
+
+import {
+  SkyAgGridRowDeleteConfirmArgs
+} from '../types/ag-grid-row-delete-confirm-args';
+
+import {
+  SkyCellType
+} from '../types/cell-type';
+
+@Component({
+  selector: 'sky-ag-grid-row-delete-component-fixture',
+  templateUrl: './ag-grid-row-delete.component.fixture.html',
+  styleUrls: ['../../../styles/ag-grid-styles.scss'],
+  encapsulation: ViewEncapsulation.None
+})
+export class SkyAgGridRowDeleteFixtureComponent implements OnInit {
+
+  public allColumnWidth: number = undefined;
+
+  public columnDefs = [
+    {
+      field: 'selected',
+      headerName: '',
+      maxWidth: 50,
+      sortable: false,
+      type: SkyCellType.RowSelector,
+      width: this.allColumnWidth
+    },
+    {
+      field: 'name',
+      headerName: 'First Name',
+      width: this.allColumnWidth
+    },
+    {
+      field: 'nickname',
+      headerName: 'Nickname',
+      editable: true,
+      type: SkyCellType.Text,
+      width: this.allColumnWidth
+    },
+    {
+      field: 'value',
+      headerName: 'Current Value',
+      editable: true,
+      type: SkyCellType.Number,
+      width: this.allColumnWidth
+    },
+    {
+      field: 'target',
+      headerName: 'Goal',
+      type: SkyCellType.Number,
+      width: this.allColumnWidth
+    },
+    {
+      field: 'date',
+      headerName: 'Completed Date',
+      editable: true,
+      type: SkyCellType.Date,
+      width: this.allColumnWidth
```

Comment:
This is just a test thing. I have a test that sets all the width to a defined value (as we had a bug in grid around this so I wanted to just carry the test over). 

In order to do that I needed a variable that I could then set in my test. I can totally tweak it to where we only set the `width` property in that one test and then reinitialize the gird but since it didn't seem to be affecting the other tests I just did it this way. It seems that if you have them all `undefined` then `ag-grid` just ignores it.